### DISCUSSION TITLE
Couldn't create repo

### DISCUSSION THREAD

**Discussion by sudhanshusingh-g** [community/community#77771] [Repositories]
I am not able to create any repo on my github account.
I am able to push previous repos commits
Whenever I am trying to create a new repo, the error says **"Owner is being deleted, so new repository cannot be created."**
I have attached the screenshot of the error for reference. I ahve also raised the ticket for this problem and waiting for the reply.
In the meanwhile , please help,if possible.
![screencapture-github-new-2023-12-02-19_30_53 (1)](https://github.com/community/community/assets/36367047/24f06a66-8731-4581-91ad-b704826e4415)

**Comment by blakealmon** [comment]
If your github account is part of an organization it may be getting deleted but otherwise your account is getting deleted for some reason so I just recommend waiting for the reply from the ticket you submitted. That would be the best route to go.

**Comment by dakshithadissanayaka** [comment]
Sure, here is a summary of the discussion and a possible solution:

**Issue:**

A user is asking why a GitHub Actions workflow that has been running successfully for a while is suddenly failing with the error message "No runners available."

**Possible Solution:**

There are a few possible reasons why a GitHub Actions workflow that was working fine is suddenly failing with the error message "No runners available." Here are some things to check:

1. **Runner capacity:** Check if there are enough runners available for your workflow. The number of runners available may have decreased due to increased demand or maintenance activities. You can view the runner capacity for your organization or repository in the "Runners" tab of your GitHub settings.

2. **Runner version:** Check if the workflow is using a runner version that is no longer supported. GitHub Actions regularly updates its runners, and older versions may be deprecated or removed over time. Ensure your workflow is using the latest recommended runner version.

3. **Runner concurrency restrictions:** Check if there are any concurrency restrictions in place for your workflow or organization. Concurrency limits control how many instances of the workflow can run simultaneously. If the limit is reached, new jobs will fail with the "No runners available" error.

4. **Workflow configuration:** Review your workflow configuration to ensure there are no errors or conflicts that might be causing the failure. Check for typos, invalid parameters, or incompatible job requirements.

5. **Resource availability:** Ensure your workflow is not requesting more resources than what is available on the runners. Excessive resource requests can lead to resource exhaustion and job failures.

6. **GitHub Actions status:** Check the GitHub Actions status page for any ongoing service disruptions or maintenance activities that could be affecting runner availability.

7. **Recent changes:** If the workflow has been working fine for a while, consider if there were any recent changes to the workflow configuration, repository code, or external dependencies. These changes might be causing the failure.

8. **Community support:** Seek help from the GitHub Actions community forums or social media platforms to see if others have encountered similar issues and if they have found solutions.

**Comment by sudhanshusingh-g** [comment]
Github support team has not updated yet. How long does it take to respond? I will start to loose my streak. How should I reach out to them?😟

**Reply by lucas-code42** [reply]
@sudhanshusingh-g I have exactly the same problem, I also opened a ticket, but I still haven't received a response from github.

**Reply by lucas-code42** [reply]
any answers @sudhanshusingh-g?

**Comment by sudhanshusingh-g** [comment]
@lucas-code42 no update. 😕 Still waiting for their reply.
Also raised another ticket with same subject just to check whether they respond or not,but no luck😞

**Comment by sudhanshusingh-g** [comment]
Hey everyone, I have just received the reply from Support team.
![image](https://github.com/community/community/assets/36367047/99fc592e-d289-42b5-a161-3fccb198def8)

I have checked and everything is working fine.
Thank you all for your help.



@lucas-code42 New Update 👆🏻

**Reply by lucas-code42** [reply]
@sudhanshusingh-g 
It worked for me too! Thanks for letting me know =]

**Comment by 0mercjh** [comment]
help !! i am facing the same issue . 
![github](https://github.com/community/community/assets/67959264/c144e528-a8d8-4396-b5d7-16bab5d9f9fb)

**Reply by sudhanshusingh-g** [reply]
Reach out to support team . Have patience it will be restored.
